docs/learn/fundamentals/contract-development/storage/state-archival.mdx

```diff
@@ -37,19 +37,23 @@ The general usage and interface are identical for all storage types. They differ
 ### `Instance`
 
 - Most expensive fees (same price as `Persistent` storage).
-- Archived when TTL goes to 0, can be restored using the [`RestoreFootprintOp`](#restorefootprintop) operation.
+- Archived when TTL goes to 0, is automatically restored via the `InvokeHostFunction` operation.
```

Comment:
I'd still leave a small note like `(or rarely using the RestoreFootprintOp)`, here and in `persistent` section